inc/modules/themeoptions/class-pdfoptions.php

```diff
@@ -1391,7 +1362,7 @@ static function getDefaults() {
 			'pdf_paragraph_separation' => 'indent',
 			'pdf_sectionopenings' => 'openauto',
 			'pdf_toc' => 1,
-			'pdf_image_resolution' => '300dpi',
+			'pdf_image_resolution' => 'auto,normal',
```

Comment:
We can remove this from the defaults, can't we? Since we don't have any way to change it anymore and it's already set in the base SCSS.